content/en/account_management/billing/custom_metrics.md

```diff
@@ -1,22 +1,57 @@
 ---
-title: Custom metrics billing
+title: Custom Metrics Billing
 kind: faq
+aliases:
+  - /integrations/faq/what-standard-integrations-emit-custom-metrics/
 ---
 
 ## Overview
 
-Custom metrics refers to any metrics that are not part of the regular integration suite, for example using custom checks or API-level metrics in your application stack. Some [standard integrations][1] can potentially emit custom metrics.
+In general, [custom metrics][1] refers to any metrics that are not part of the regular integration suite, for example using custom checks or API-level metrics in your application stack.
 
-Pro plans include 100 custom metrics per host and Enterprise plans include 200 custom metrics per host. This number is averaged across all paid hosts. Additional custom metrics packages can be purchased.
+* Billable count is based on average number of `custom metrics / hour` for the month.
+* Pro plans include 100 custom metrics per host.
+* Enterprise plans include 200 custom metrics per host.
+* The number of metrics is averaged across all paid hosts.
+* Additional custom metrics packages can be purchased.
```

Comment:
Should we link to sales@datadoghq.com here?

Reply:
Sales is linked in the line below.